modules/cache-common/src/main/java/org/opensearch/cache/common/tier/TieredSpilloverCacheStatsHolder.java

```diff
@@ -107,18 +107,19 @@ public void incrementMisses(List<String> dimensionValues) {
 
     @Override
     public void incrementEvictions(List<String> dimensionValues) {
-        final String tierValue = validateTierDimensionValue(dimensionValues);
+        incrementEvictions(dimensionValues, true);
```

Comment:
In `TieredSpilloverCache` we are using the below overloaded `incrementEvictions` method and looks like that is what the intent is. So we should just throw `UnsupportedOperationException` from this method to avoid any wrong usage in future. Lets also add a java doc for this method explaining why other one should be used for Tiered cache.